dnf-behave-tests/fixtures/repos.d/repo-sync-errors-mirrorlist-bad.html

```diff
@@ -0,0 +1,3 @@
+file:///nonexistent.repo
+http://127.0.0.1:5000/nonexistent
+
```

Comment:
These files in `fixtures/repos.d` are no longer needed, right? You forgot to drop them from the commit.

Reply:
Right, thanks!